src/main/util.ts

```diff
@@ -105,10 +114,8 @@ const loadAllVideos = (storageDir: any, videoState: any) => {
 /**
  * Get the date a video was recorded from the date object.
  */
-const getMetadataForVideo = (video: any) => {
-    const videoFileName = path.basename(video.name, '.mp4');
-    const videoDirName = path.dirname(video.name);
-    const metadataFile = videoDirName + "/" + videoFileName + ".json";
+const getMetadataForVideo = (video: string) => {
+    const metadataFile = getMetadataFileForVideo(video)
```

Comment:
My VSCode is upset with line 122 here, not sure if it's as a result of this change:
`Argument of type 'Buffer' is not assignable to parameter of type 'string'`

Reply:
I think the fix is simply to add `.toString()` on the end. Would you sort that as you're in the area? I think it's pre-existing.

Reply:
`const metadata = JSON.parse(metadataJSON.toString());` 

https://stackoverflow.com/questions/46190511/typescript-error-ts2345-error-ts2345argument-of-type-buffer-is-not-assignabl

Reply:
Yeah, it was always there - I'll include it.